iam_groups_authn/utils.py

```diff
@@ -33,3 +34,33 @@ async def run(*args, loop=None, executor=None, **kwargs):
         return await loop.run_in_executor(executor, pfunc)
 
     return run
+
+
+class DatabaseVersion(Enum):
+    """Enum class for database version."""
+
+    MYSQL_8_0 = "mysql"
+    POSTGRES_13 = "postgresql"
+    POSTGRES_12 = "postgresql"
+    POSTGRES_11 = "postgresql"
+    POSTGRES_10 = "postgresql"
+    POSTGRES_9_6 = "postgresql"
+
+
+class RoleService:
```

Comment:
Can you add a comment here describing the purpose for these classes? 

This should also probably use [ABC](https://docs.python.org/3/library/abc.html#abc.ABC) to make sure we are enforcing this interface at compile